src/common/lambdaConfig.go

```diff
@@ -0,0 +1,92 @@
+package common
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"gopkg.in/yaml.v3"
+)
+
+var LambdaConf *LambdaConfig
+
+type HTTPTrigger struct {
+	Method string `yaml:"method"` // HTTP method (e.g., GET, POST)
```

Comment:
Can we have a "*" option (for all methods) and default to that?

Reply:
Once we add this config, we should allow HTTP requests (or not) based on the HTTP trigger config.

The general HTTP handler is RunLambda, and it gets a function from the Lambda Manager here: https://github.com/open-lambda/open-lambda/blob/main/src/worker/event/lambdaServer.go#L62

Before invoking it, we should check the config for that lambda: does it have an HTTP trigger corresponding to the Request Method?  https://pkg.go.dev/net/http#Request.  If not, we should return an error.